py/dml/dmlc.py

```diff
@@ -502,11 +508,13 @@ def show_illegal_attributes(option, opt, value, parser):
             else:
                 f = open(options.makedep, 'w')
             with f:
-                # rebuild from DML if imported file has changed
-                f.write('%s %s.c : %s\n' % (
-                    options.makedep, outputbase, deps))
-                # if imported file vanished, survive and rebuild
-                f.write('%s :\n' % (deps,))
+                    # rebuild from DML if imported file has changed
+                if options.depninja:
+                    f.write('%s.c : %s\n' % (outputbase, deps))
```

Comment:
Following original code here. If `pathlib.Path(..).write_text(..)` is preferable, it should be a separate PR.

Reply:
Agreed (it's preferrable, and a separate PR)